src/Vgrep/Widget/Pager/Internal.hs

```diff
@@ -24,7 +26,7 @@ data Pager = Pager
     { _column      :: Int
     -- ^ The current column offset for horizontal scrolling
 
-    , _highlighted :: Set Int
+    , _highlighted :: IntMap (Formatted Attr)
     -- ^ Set of line numbers that are highlighted (i.e. they contain matches)
 
     , _above       :: Seq Text
```

Comment:
I think it's not even enabled. I just kept the alignment when adding Haddocks...